app/jobs/close_description_change_job.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class CloseDescriptionJob < ApplicationJob
+  queue_as :default
+
+  def self.description_change_requests
+    DescriptionChangeValidationRequest.open_change_created_over_5_days_ago
+  end
+
+  def perform
+    description_change_requests.each do |change_request|
+      auto_update_and_notify(change_request)
+    end
+  end
+
+  def auto_update_and_notify(description_change_request)
+    description_change_request.planning_application.update!(description: description_change_request.proposed_description)
+    description_change_request.update!(state: "closed", approved: true)
```

Comment:
We don't want to update the state manually, that's what we hired `aasm` for. Instead I suggest we reopen AASM in `DescriptionChangeValidationRequest` and do something like:

```ruby

  aasm do
    event :approve! do
      after do |description|
        update!(proposed_description: description)
      end

      transitions from: :open, to: :closed
    end

    event :reject! do
      after do
        update!(rejection_reason: I18n.t("some localised message"))
      end

      transitions from: %i[pending open], to: :closed
    end
  end
  ```